backend/app/consts.py

```diff
@@ -131,6 +136,8 @@ class PanelId(str, Enum):
     "r": ["popularity desc"],
     "pdmr": ["start_date desc"],
     "pdlr": ["start_date asc"],
+    "cdmr": ["created_at desc"],
+    "cdlr": ["created_at asc"],
```

Comment:
Date sorting got out of hand here. 
- here we have 6 args for sorting by date only `"cdmr", "cdlr", "pdmr", "pdlr", "dmr", "dlr"`
- definitely, I would unify `created_at` from catalogs and name it `publication_date` as it is in other data types. @wiktorflorian could you please do that? 
- `pdmr, pdlr` we can even consider renaming "`start_date`" -> `publication_date` if that makes sense

Reply:
I will rename it, on my end.

Reply:
please ping @maria-j-k when ready

Reply:
@wiktorflorian 
- lets rename only `created_at` -> `publication_date` in catalogues
- lets leave `start_date` as it is in projects & orgs